src/SafeDNS/TemplateRecordClient.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @author: John Birch-Evans <john.birch-evans@ukfast.co.uk>
+ * @copyright: 2020 UKFast.net Ltd
+ */
+
+namespace UKFast\SDK\SafeDNS;
+
+use UKFast\SDK\Client;
+use UKFast\SDK\SafeDNS\Entities\Record;
+
+class TemplateRecordClient extends Client
+{
+    const RECORD_MAP = [
+        'created_at' => 'createdAt'
+    ];
+
+    /**
+     * Gets a paginated list of records for a template
+     * @param $id
+     * @param int $page
+     * @param int $perPage
+     * @param array $filters
+     * @return int|\UKFast\SDK\Page
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getPage($id, $page = 1, $perPage = 15, $filters = [])
```

Comment:
To match the rest of the SafeDNS SDK clients, please could this `per_page` default be 20?

```suggestion
    public function getPage($id, $page = 1, $perPage = 20, $filters = [])
```

Reply:
There are a lot of places that set this to 15 by default. 15 is also the default for our API, as seen in our api guide:

![image](https://user-images.githubusercontent.com/7416827/81178072-8d373900-8f9f-11ea-9205-7b25def642bf.png)

https://developers.ukfast.io/api-guide